docs/index.md

```diff
@@ -143,8 +143,10 @@ fundamentals/flow/index
 fundamentals/gateway/index
 fundamentals/client/client
 fundamentals/k8s
+fundamentals/docarray-support/index
```

Comment:
I would not call `docarray-support` but `dependency`?

Reply:
this is just the folder. I renamed it to `docarray-dependency` since it is just for docarray.

The title of the section is `coming from docarray` because it is using the title on the md files rather than the fodler name